### PR TITLE
interactive bots: Run the helloworld bot for KubsHD.

### PR DESCRIPTION
**Pull request by KubsHD**

### CODE REVIEW

File: interactive-bots/helloworld/KubsHD/notes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The only problem with this project, was that the bot crashed for some unknown reason, tools/do-destroy-rebuild-database fixed it.
```
Comment (tommyip) [inline interactive-bots/helloworld/KubsHD/notes.md]: Can you line wrap all text files to 79 characters?

**Review by tommyip — COMMENTED** [review]
Seems like you are creating commits directly from Github? We strongly suggest you do this with git in a terminal or a git GUI tool, so that you can update your commits in the future.

For future reference, please **DON'T** open multiple PRs for a single task.

File: interactive-bots/helloworld/KubsHD/notes.md (hunk 1 above)
Comment (abhijeetkaur) [reply interactive-bots/helloworld/KubsHD/notes.md]: @KubsHD You are requested to break this line into two for better readability. :)

Comment (KubsHD) [reply interactive-bots/helloworld/KubsHD/notes.md]: Okay, I changed it.

**Pull request merged**